modular_splurt/code/modules/jobs/job_types/_job_alt_titles.dm

```diff
@@ -337,7 +449,20 @@
 		"Hunter",
 		"Ashwalker Sex Slave",
 		"Ashwalker Breeder",
-		"Slayer"
+		"Slayer",
+		"Robustinator"
 	)
 	LAZYADD(alt_titles, extra_titles)
 	. = ..()
+	
+	
+// Silicon
+/datum/job/ai
+	alt_titles = list(
+		"Sample Alt Title"
+	) // ZERO CLUE if this will work, need to gametest
+	
+/datum/job/cyborg
+	alt_titles = list(
+		"Automaton"
+	) 
```

Comment:
please add extra titles using LAZYADD and New() like in the other classes if they're NOT SPLURT-original jobs